pipenv/environment.py

```diff
@@ -476,22 +465,17 @@ def get_include_path(self) -> Optional[Dict[str, str]]:
         :return: The python include path for the environment
         :rtype: Dict[str, str]
         """
-        tmpfile = vistir.path.create_tracked_tempfile(suffix=".json")
-        tmpfile.close()
-        tmpfile_path = make_posix(tmpfile.name)
         py_command = (
-            "import distutils.sysconfig, io, json, sys; paths = {{u'include': "
-            "u'{{0}}'.format(distutils.sysconfig.get_python_inc(plat_specific=0)), "
-            "u'platinclude': u'{{0}}'.format(distutils.sysconfig.get_python_inc("
-            "plat_specific=1)) }}; value = u'{{0}}'.format(json.dumps(paths));"
-            "fh = io.open('{0}', 'w'); fh.write(value); fh.close()"
+            "import distutils.sysconfig, io, json, sys; paths = {u'include': "
```

Comment:
I realize you are just refactoring existing code, but now I notice this usage of distutils which is totally deprecated.  I wonder what the correct transformation of this usage would be.

Reply:
Instead of writing to a file, I refactor so it print out to the `stdout` directly. 
Because the filename should not be passed to string as variable anymore, I deleted some unnecessary curly braces.

Reply:
As you said, I tried to remove distutils.sysconfig and use sysconfig instead.

Reply:
Since `sysconfig.get_path` does not Exception like `distutils.sysconfig`, the `c.returncode` is always `0`. Therefore, those lines below could be refactored
https://github.com/pypa/pipenv/blob/75b77ebebf5559c75f3ece9eba068afecba3769b/pipenv/environment.py#L393-L415